manifests/0000_70_cluster-network-operator_01_pki_crd.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: pkis.network.operator.openshift.io
+spec:
+  group: network.operator.openshift.io
+  names:
+    kind: PKI
```

Comment:
I'm going to rename this to something more obscure. I don't want people discovering `oc create pki`